docs/sources/alerting/fundamentals/alert-rules/annotation-label.md

```diff
@@ -100,15 +98,16 @@ Here is an example that shows how to exclude the label `Team`. You can choose be
 
 An alert's label set can contain three types of labels:
```

Comment:
i corrected these to alert rule - but i'm not sure anymore. is it alert rule, or alert instance? or...?

Reply:
alert rule is correct 😃